src/db/collections/outputs/mod.rs

```diff
@@ -48,24 +48,64 @@ pub struct OutputDocument {
 
 /// The stardust outputs collection.
 pub struct OutputCollection {
-    db: mongodb::Database,
+    db: MongoDb,
```

Comment:
Why make this change?

Reply:
I only made this change due to what you said here:
https://github.com/iotaledger/inx-chronicle/pull/779#discussion_r990251965
:shrug: